client/my-sites/site-settings/comment-display-settings/index.jsx

```diff
@@ -0,0 +1,85 @@
+/**
+* External dependencies
+*/
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormSelect from 'components/forms/form-select';
+import FormTextInput from 'components/forms/form-text-input';
+import JetpackModuleToggle from '../jetpack-module-toggle';
+import { isJetpackModuleActive } from 'state/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+class CommentDisplaySettings extends Component {
+	shouldEnableSettings() {
+		const { isCommentsModuleActive, submittingForm } = this.props;
+		return !! submittingForm || ! isCommentsModuleActive;
+	}
+
+	render() {
+		const {
+			fields,
+			onChangeField,
+			selectedSiteId,
+			submittingForm,
+			translate,
+		} = this.props;
+
+		return (
+			<FormFieldset className="comment-display-settings">
+				<JetpackModuleToggle
+					siteId={ selectedSiteId }
+					moduleSlug="comments"
+					label={ translate(
+						'Use Jetpack comments. Let readers use their WordPress.com, Twitter, Facebook or Google+ to leave comments ' +
+						'on your posts and pages.'
+					) }
+					disabled={ !! submittingForm }
+					/>
+				<div className="comment-display-settings__module-setting is-indented">
+					<FormLabel htmlFor="highlander_comment_form_prompt">{ translate( 'Comments Label' ) }</FormLabel>
+					<FormTextInput
+						name="highlander_comment_form_prompt"
+						type="text"
+						id="highlander_comment_form_prompt"
+						value={ fields.highlander_comment_form_prompt || '' }
+						onChange={ onChangeField( 'highlander_comment_form_prompt' ) }
+						disabled={ this.shouldEnableSettings() } />
+					<FormSettingExplanation>
+						{ translate( 'A few catchy words to motivate your readers to comment.' ) }
+					</FormSettingExplanation>
+				</div>
+				<div className="comment-display-settings__module-setting is-indented">
+					<FormLabel htmlFor="jetpack_comment_form_color_scheme">{ translate( 'Color Scheme' ) }</FormLabel>
+					<FormSelect
```

Comment:
@MichaelArestad this one's implemented with radio buttons in Jetpack. I'd suggest being consistent, but which one would you recommend in that particular case - select or radio?